include/lsst/jointcal/PhotomFit.h

```diff
@@ -17,113 +17,109 @@ namespace jointcal {
 
 class Associations;
 
-/*! Some comments.
-
-
-*/
-
-
 //! Class that handles the photometric least squares problem.
-
 class PhotomFit {
   private :
 
-  Associations &_assoc;
-  std::string _WhatToFit;
+  Associations &_associations;
+  std::string _whatToFit;
   bool _fittingModel, _fittingFluxes;
   unsigned _nParModel, _nParFluxes, _nParTot;
   PhotomModel * _photomModel;
   double _fluxError;
-  int _LastNTrip; // last triplet count, used to speed up allocation
-
-
+  int _lastNTrip; // last triplet count, used to speed up allocation
 
  public :
 
   //! this is the only constructor
-  PhotomFit (Associations &A, PhotomModel *M, double PosError);
+  PhotomFit (Associations &associations, PhotomModel *model, double fluxError);
 
   //! Does a 1 step minimization, assuming a linear model.
   /*! It calls AssignIndices, LSDerivatives, solves the linear system
     and calls OffsetParams. No line search. Relies on sparse linear
     algebra. */
-  bool Minimize(const std::string &WhatToFit);
+  bool minimize(const std::string &whatToFit);
```

Comment:
I've cleared up these documents to some degree: fortunately, there was a description of whatToFit in the .cc file for both AstromFit and PhotomFit. I already filed a ticket about this parameter: https://jira.lsstcorp.org/browse/DM-8046